components/dashboard/src/workspaces/VideoCarousel.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import React, { useState } from "react";
+import { trackVideoClick } from "../Analytics";
+
+import "lite-youtube-embed/src/lite-yt-embed.css";
+import "lite-youtube-embed/src/lite-yt-embed";
+
+interface Video {
+    id: string;
+    title: string;
+    analyticsLabel: string;
+}
+
+const videos: Video[] = [
+    { id: "1ZBN-b2cIB8", title: "Gitpod in 120 seconds", analyticsLabel: "gitpod-demo" },
+    { id: "zhZNnzFlZnY", title: "Getting started with Gitpod", analyticsLabel: "getting-started-with-gitpod" },
+    { id: "kuoHM2bpBqY", title: "Fully automate your dev setup", analyticsLabel: "automate-gitpod-setup" },
+    { id: "_CwFzCbAsoU", title: "Personalise your workspace", analyticsLabel: "personalise-gitpod-workspace" },
+];
+
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            "lite-youtube": any;
+        }
+    }
+}
+
+export const VideoCarousel: React.FC = () => {
+    const [currentVideo, setCurrentVideo] = useState(0);
+
+    const handleDotClick = (index: number) => {
+        setCurrentVideo(index);
+    };
+
+    const onPlayerStateChange = (index: number) => {
+        trackVideoClick(videos[index].analyticsLabel);
+    };
+
+    return (
+        <div className="video-carousel">
+            <div className="video-container">
+                {videos.map((video, index) => (
+                    <div key={video.id} style={{ display: index === currentVideo ? "block" : "none" }}>
+                        <lite-youtube
+                            videoid={video.id}
+                            style={{
+                                width: "320px",
+                                height: "180px",
+                            }}
+                            class="rounded-lg"
+                            playlabel={video.title}
+                            onClick={() => onPlayerStateChange(index)}
+                        ></lite-youtube>
+                    </div>
+                ))}
+            </div>
+            <div className="flex justify-center space-x-2 mt-2">
+                {videos.map((_, index) => (
+                    <button
```

Comment:
nit: it's somewhat confusing to switch between videos with just these ellipses. If I wanted to find a video among these, I would have to go through each one.

Reply:
What alternative UX design would you suggest to fix this?

Reply:
I think we would have to reconsider this entire section (a proper and informed carousel wouldn't fit), where we'd have the video title or a keyword relevant to the video visible for every one. Most likely not the effort at this point, though

Reply:
Thanks Filip! Can be a follow up issue, I don't think it's a major problem right now that should block the release

Reply:
Agree it's no blocker 👍